doc/whats_new.rst

```diff
@@ -23,6 +23,8 @@ Changelog
 
     - Add label center of mass function :func:`mne.Label.center_of_mass` by `Eric Larson`_
 
+    - Added :func:`mne.viz.ica.plot_properties` that allows ploting of independent component properties similar to ``pop_prop`` in EEGLAB. Also :class:`mne.preprocessing.ica.ICA` has ``plot_properties`` method now. Added by `Mikołaj Magnuski`_
```

Comment:
It is called `plot_properties` because it will be able to plot channel properties (soon, in another PR). Should I make a separate function that is called `plot_ica_properties` that uses `_plot_properties` (the current function made private) for example?